Monika After Story/game/script-stories.rpy

```diff
@@ -1809,3 +1809,62 @@ label mas_scary_story_flowered_lantern_3:
     m 2wfc "And lying beside him in the bed were the bones of a woman with its arms clung around his neck as if in an embrace."
     call mas_scary_story_cleanup
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_story_database,
+            eventlabel="mas_scary_story_clever_girl",
+            category=[store.mas_stories.TYPE_SCARY],
+            prompt="Clever girl",
+            unlocked=False
+        ),
+    code="STY"
+    )
+
+label mas_scary_story_clever_girl:
+    call mas_scary_story_setup
+    m 1eua "A 15-year old girl named Donna lived with her father in a small house in the suburbs."
+    m 1eka "Ever since her mother died, Donna had depended on her father for everything. They had a wonderful relationship and loved each other very much."
+    m 2euc "One morning, Donna's father was leaving on a business trip. As they ate breakfast together, he told her that he would be home very late that night."

```

Comment:
`    m 2euc "One morning, Donna's father was leaving on a business trip. As they ate breakfast together, he told her that he would be home very late that night. {w=0.2}{nw}"`